plugins/default/lib/disk_usage.py

```diff
@@ -53,6 +62,10 @@ def analize(pod, disks, warning_threshold, critical_threshold):
 
 
 def report(results):
+    # REVIEW: is nagios.UNKNOWN the best here?
```

Comment:
I think nagios.UNKNOWN makes the most sense, but I am not against changing it if somebody has a good argument otherwise.